src/esbmc/bmc.cpp

```diff
@@ -910,17 +924,36 @@ smt_convt::resultt bmct::multi_property_check(
       else
         reached_claims.emplace(cmt_loc);
 
-      // Generate Output
-      std::string output_file = options.get_option("cex-output");
-      if (output_file != "")
+      // for verbose output of cond coverage
+      if (is_vb)
       {
-        std::ofstream out(fmt::format("{}-{}", ce_counter++, output_file));
-        show_goto_trace(out, ns, goto_trace);
+        auto current_pair = std::make_pair(claim.claim_msg, claim.claim_loc);
+        if (total_cond.count(current_pair))
+        {
+          if (options.get_bool_option("condition-coverage-claims-vb"))
```

Comment:
What is `vb`?

Reply:
It's for the runtime **v**erbose **o**utput. By enabling this, we will output the coverage information when handling each instrumentation assertion.  This will be helpful when the program is too large and ESBMC cannot finish, we can still get some info about the coverage.

Will add documentation